compute/cloud-client/src/test/java/compute/disks/HyperdisksIT.java

```diff
@@ -96,15 +96,15 @@ public void stage1_CreateHyperdiskTest()
     Assert.assertTrue(hyperdisk.getZone().contains(ZONE));
   }
 
-  @Disabled
   @Test
   public void stage1_CreateHyperdiskStoragePoolTest()
```

Comment:
why the name of the method follows snake notation and not pascal notation?

Reply:
It's not a scope of this task. I've created a new task to fix code and will follow yours recommendations.

Reply:
Fixed